rosbag2_transport/src/rosbag2_transport/recorder.cpp

```diff
@@ -218,8 +201,15 @@ std::string Recorder::serialized_offered_qos_profiles_for_topic(const std::strin
   return YAML::Dump(offered_qos_profiles);
 }
 
-rclcpp::QoS Recorder::common_qos_or_fallback(const std::string & topic_name)
+rclcpp::QoS Recorder::subscription_qos_for_topic(const std::string & topic_name)
 {
+  rclcpp::QoS subscription_qos{10};
```

Comment:
Same comment regarding the QoS default. `10` is used in multiple places, use a common default instead (no magic numbers)?